sklvq/models/_gmlvq.py

```diff
@@ -386,8 +386,8 @@ def _normalize_omega(omega: np.ndarray) -> None:
 
     def _correct_omega(self, omega: np.ndarray) -> None:
         # Note matmul is faster for larger matrices, for smaller dot is faster.
-        np.matmul(omega, self.relevance_correction.T, out=omega)
-        #np.matmul(omega.T, self.relevance_correction, out=omega.T)
+        np.matmul(self.relevance_correction,omega,out=omega.T)
+        # will this still keep omega as a row matrix when there is a omega.T in out???????????????????????????????????????????????????        
```

Comment:
Don't know. You could test this. Are you certain the transposes are now all correct? You changed them again. It would also depend on the correction matrix, are you sure that is correct?